### ISSUE TITLE
emulate touchscreen not work in 3.1

### ISSUE THREAD

**Issue by alexzheng** [godotengine/godot#21642]
**Godot version:** 3.1 alpha1

**OS/device including version:**

**Issue description:**

check Emulate TouchScreen on will not generator InputEventScreen* event when mouse dragged or pressed.

**Steps to reproduce:**

**Minimal reproduction project:**

**Comment by RandomShaper** [comment]
For 3.1 I changed the path to input emulation settings and their name so they are more intuitive. They are now `input_devices/pointing/emulate_touch_from_mouse` and `input_devices/pointing/emulate_mouse_from_touch`.

If you have the setting from 3.0, it will still appear in project settings, but with no effect on 3.1.

I was a bit concerned about the change because users may get confused, so I opened #18092. Other issues relevant here are #20626 and #18758.

At this point I only can see the solution of converting the settings when opening a 3.0 project in 3.1. That, along with the compatibility warning displayed as discussed in the referred PRs would provide the best experience.

Has any other setting been renamed/moved?
